### DISCUSSION TITLE
LXQt Power Management icons?

### DISCUSSION THREAD

**Discussion by 097115** [lxqt/lxqt#2341] [General]
Hi!

Where are the icons from the power management settings located? `Circle`, `Circle with percentage`, `Battery with percentage opaque`, etc (NOT the theme icons)?

Or they aren't available as separate graphic files, I suspect? :)

**Comment by palinek** [comment]
You're right. Those graphics are generated on the fly by the code.

**Reply by tsujan** [reply]
Also, LXQt Power Management has an option for using system icons. But the hard-coded graphics gives more info.
